packages/docs/src/pages/contact.module.css

```diff
@@ -1,28 +1,70 @@
+.preventfulContainer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+}
 .container {
   display: flex;
   flex-direction: column;
   gap: 24px;
-}
+  width: 1280px;  
+  padding: 0 40px;
 
+}
+@media (min-width: 1200px) {
+  .container {
+    display: flex;
+    flex-direction: column;
+    gap: 24px;
+    padding: 0 40px;
```

Comment:
according to designs the left & right padding should be 24px on mobile
<img width="646" height="392" alt="image" src="https://github.com/user-attachments/assets/42e42327-c121-4d18-aaa9-c42f5ca091b1" />